.github/modify_sitemap.js

```diff
@@ -34,7 +34,7 @@ fs.readFile(inputFile, "utf8", (err, data) => {
   const urlBlockPattern = /(<url>[\s\S]*?<\/url>)/g;
 
   let filteredContent = newContent.replace(urlBlockPattern, (match) => {
-    if (/https:\/\/deriv\.com\/eu\//.test(match)) {
+    if (/https:\/\/deriv\.com.*\/eu\//.test(match)) {
```

Comment:
```suggestion
    if (/https:\/\/deriv\.com(\/[a-z-]{2,5})?\/eu\//.test(match)) {
```

Reply:
This is more accurate @moiz-deriv .
It matches `deriv.com/eu/...`
or `deriv.com/zh-cn/eu/...`
or `deriv.com/es/eu/...`

Reply:
@ali-hosseini-deriv but this will fail for https://deriv.com/zh-cn/test/eu/ 
*\/eu will check for all strings as long as there is a /eu/ the url will be removed from the sitemap